src/DssSpellCollateral.sol

```diff
@@ -0,0 +1,179 @@
+// SPDX-FileCopyrightText: © 2021 Dai Foundation <www.daifoundation.org>
+// SPDX-License-Identifier: AGPL-3.0-or-later
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity 0.6.12;
+
+import "dss-exec-lib/DssExecLib.sol";
+
+contract DssSpellCollateralAction {
+    // --- Math ---
+    uint256 constant RAY  = 10 ** 27;
+
+    // --- Offboarding: Current Liquidation Ratio ---
+    uint256 constant CURRENT_UNI_A_MAT              =  165 * RAY / 100;
+    uint256 constant CURRENT_UNIV2DAIETH_A_MAT      =  120 * RAY / 100;
+    uint256 constant CURRENT_UNIV2WBTCETH_A_MAT     =  145 * RAY / 100;
+    uint256 constant CURRENT_UNIV2WBTCDAI_A_MAT     =  120 * RAY / 100;
+
+    // --- Offboarding: Target Liquidation Ratio ---
+    uint256 constant TARGET_UNI_A_MAT               = 1_300 * RAY / 100;
+    uint256 constant TARGET_UNIV2DAIETH_A_MAT       = 2_000 * RAY / 100;
+    uint256 constant TARGET_UNIV2WBTCETH_A_MAT      = 2_400 * RAY / 100;
+    uint256 constant TARGET_UNIV2WBTCDAI_A_MAT      = 800 * RAY / 100;
+
+    // --- Rates ---
+    // Many of the settings that change weekly rely on the rate accumulator
+    // described at https://docs.makerdao.com/smart-contract-modules/rates-module
+    // To check this yourself, use the following rate calculation (example 8%):
+    //
+    // $ bc -l <<< 'scale=27; e( l(1.08)/(60 * 60 * 24 * 365) )'
+    //
+    // A table of rates can be found at
+    //    https://ipfs.io/ipfs/QmTRiQ3GqjCiRhh1ojzKzgScmSsiwQPLyjhgYSxZASQekj
+    //
+
+    function onboardCollaterals() internal {
```

Comment:
just a note that this function is called `onboardNewCollaterals` in goerli.  We should either patch goerli or make this function name conform.  Either way, they should use the same name.

Reply:
done in https://github.com/makerdao/spells-mainnet/pull/258/commits/a702255a02e56acaac428e47d9d6e2a723c7ba26